### PR TITLE
Moved sample bots and modified script for Travis CI

### PR DESCRIPTION
**Pull request by pkhdipraja**
As the repository is intended for bots only, I commented out the server and documentation section for Travis CI.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+dist: xenial
```
Comment (janagoetze) [inline .travis.yml:1]: I can leave a comment

File: .travis.yml [hunk 2]
```diff
@@ -37,18 +37,18 @@ jobs:
...
+#     script: (cd docs && make html)
+#     after_success: touch docs/_build/html/.nojekyll
+#     deploy:
+#       provider: pages
+#       skip-cleanup: true
+#       local_dir: docs/_build/html
+#       github-token: $GH_TOKEN
+#       keep-history: true
+#       on:
+#         branch: master
```
Comment (janagoetze) [inline .travis.yml:54]: I think that if we do not need this, then we should just remove it

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+jobs:
+  include:
+  - stage: deploy
+    name: Docker
+    language: ruby
+    services:
+    - docker
+    script:
+    #- docker build -t slurk/server -f docker/slurk/Dockerfile .
```
Comment (janagoetze) [inline .travis.yml:16]: what about this (and other) commented line? do we need it? if yes, let's add a comment, if no, I think it should go

Comment (pkhdipraja) [reply .travis.yml:16]: I don't think this and the other commented line are needed, however I prefer to comment it out first as I am not that familiar with Travis.

Comment (janagoetze) [reply .travis.yml:16]: ok, let's remove the comments because version control saves them for us. we can go back to them if necessary

Comment (pkhdipraja) [reply .travis.yml]: I updated the commit with the comments removed.

**Pull request merged**